docs/src/components/footer.js

```diff
@@ -1,55 +1,106 @@
 import React from 'react'
 import styled from 'styled-components'
+import { Flex, Box } from '@rebass/grid'
+import { height, space } from 'styled-system'
 
 import { PanelProvider, Wrapper } from './theme'
-import dta from '../images/dta-wordmark.png'
+import dta from '../images/dta-wordmark-white.svg'
+import { External } from './link'
 
 const Logo = styled.img`
-  height: 4rem;
+  padding-left: 0;
+  margin-bottom: 2rem;
+  ${height};
 `
 
 const Root = styled.footer`
   background: ${props => props.theme.background};
   color: ${props => props.theme.content};
+  padding-top: 2rem;
+  padding-bottom: 5rem;
 `
 
-const Columns = styled.div`
+const StyledFlex = styled(Flex)`
+  flex-wrap: wrap;
   margin: 4rem 0;
-  display: flex;
-  justify-content: start;
+
+  justify-content: space-between;
   align-items: start;
+  border-bottom: 1px solid #bfc1c3;
+  padding-bottom: 4rem;
 
-  & > * + * {
-    margin-left: 4rem;
-  }
+  ${space};
 `
 
 const Ul = styled.ul`
+  padding-left: 0;
   margin-top: 0;
   margin-bottom: 0;
+  list-style-type: none;
 `
 
+const UnorderedList = ({ list }) => (
+  <Box width={[1, 1 / 4]}>
+    <Ul>
+      {list.map(i => (
+        <li key={i.text}>
+          <External blank href={i.target}>
```

Comment:
I think we can leave `blank` off - user can open in new window if they want.

Reply:
Removed